test/unit/__mocks__/@wordpress/data.js

```diff
@@ -0,0 +1,3 @@
+import { use, plugins } from '../../../../packages/data/src';
+use( plugins.asyncGenerator );
```

Comment:
What thing is depending on async generators in the test? Should they be, or should those tests be part of the suite for the `asyncGenerator` plugin instead?

Reply:
This is needed for some `effects` tests. We have some effects triggering async generator resolvers that won't work without this.

Reply:
This is something that won't be needed in the future if we rewrite those effects as generators with controls as we can directly feed the generator with mocked results in the tests without running the controls.